src/mudlet.pro

```diff
@@ -1433,14 +1433,18 @@ macx {
 }
 
 win32 {
-    # set the Windows binary icon
+    # set the Windows binary icon, the proper .ico file should contains several
+    # layers of increasing sizes: 16, 24, 32, 48, 64, 128, 256 - as well as
+    # being used for the executable application file when it is displayed in
+    # the Windows explorer it is also used in some other places (installed
+    # applications in Windows settings/control panel)
```

Comment:
Comment creep explaining basic icon files' dynamics. 
Is that really needed in Mudlet source?
Isn't there a wiki or something?

Reply:
In the *mudlet/installer* repository we have a `./windows/mudlet_main_48px.ico` which is a single 48x48 image (and a `./windows/mudlet_main_512x512px.png`).

That 48x48 icon is being used for the Windows `setup.exe` currently - and that DOES need revising.

🤔 TBH I don't tend to go to the Wiki whilst hacking the codebase - but it would be useful to store these details somewhere.

Reply:
Agree on moving this out, perhaps good to start making use of the wiki?

Reply:
Done - and moved to https://wiki.mudlet.org/w/Notes_on_QMake_variables .